fungible_conversion_proxy/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "fungible_conversion_proxy"
+version = "0.0.1"
+authors = ["Request Labs"]
```

```suggestion
authors = ["Request Network Foundation"]
```
Just a detail but since the bounty is subsidised by the foundation, it's more relevant like this.